VenafiPS/Public/New-TppCapiApplication.ps1

```diff
@@ -204,8 +204,9 @@ function New-TppCapiApplication {
         if ( -not $PSBoundParameters.ContainsKey('SkipExistenceCheck') ) {
 
             if ( $PSBoundParameters.ContainsKey('CertificatePath') ) {
-                $certPath = (Split-Path $CertificatePath -Parent)
-                $certName = (Split-Path $CertificatePath -Leaf)
+                $certName = (Split-Path -Path $CertificatePath -Leaf)
+                $certPath -replace ('\\+{0}' -f $certName), ''
```

Comment:
Should this line be changed to assign the value to certPath ?  The updates from the -replace are not stored anywhere.

Reply:
good catch, thanks.  had this in a commit I hadn't pushed.  as I was looking into this, I found one other split-path in this file that needed to change as well